src/tests/ForecastDetails.test.js

```diff
@@ -1,43 +0,0 @@
-import React from "react";
-import { render } from "@testing-library/react";
-import ForecastDetails from "../components/ForecastDetails";
-
-describe("ForecastDetails", () => {
-  const validProps = {
-    date: 1111111,
-    description: "Stub description",
-    icon: "800",
-    humidity: 14,
-    temperature: {
-      max: 22,
-      min: 12,
-    },
-    wind: {
-      speed: 13,
-      direction: "s",
-    },
-  };
-  it("renders correctly", () => {
-    const { asFragment } = render(<ForecastDetails forecast={validProps} />);
-    expect(asFragment()).toMatchSnapshot();
-  });
-  it("renders correct values for props", () => {
-    const { getByText } = render(<ForecastDetails forecast={validProps} />);
-    expect(getByText("Date:Thu Jan 01 1970")).toHaveClass(
```

Comment:
`toHaveClass` isn't something you would want to use very often for react testing because the user isn't going to know or care what the class is. Try checking for the actual content of one of your components using `toHaveTextContent`